### DISCUSSION TITLE
Feature importance- COPOD, others.

### DISCUSSION THREAD

**Discussion by anirbanmukherjee2709** [yzhao062/pyod#436] [General]
Hi @yzhao062 

I am looking to enumerate feature importance for features that lead a case to become an anomaly, for my project. I have extracted the _Isolation_forest_ importance. While COPOD just returns the plot. Have you considered:

1. returning the feature importance as you mention in commented-out codes?
2. any other algorithms to have the explainability methods? 

Thanks in advance!

**Comment by duvi86** [comment]
Hi all, I have the same issue, I would like to see from where potentially the anomaly is coming. Is there something like that?

**Reply by onixlas** [reply]
Well, ECOD and HBOS have a built-in explainability.
